Source/ChilliSource/Core/Base/Application.cpp

```diff
@@ -508,7 +556,7 @@ namespace ChilliSource
     //------------------------------------------------------------------------------
     void Application::Render() noexcept
     {
-        m_renderer->ProcessRenderCommandBuffer();
+        m_renderer->ProcessRenderCommandBuffers();
```

Comment:
Why has this changed? Only one buffer should be processed each time this is called, otherwise it no longer mirrors how Vulkan works.